apps/tehwolfde/src/app/i18n/language-switcher.component.scss

```
@@ -7,16 +7,23 @@
-  padding: 0 8px;
+  min-width: 48px;
+  height: 48px;
+  padding: 0 10px;
 }
 
 // Material wraps projected content in .mdc-button__label, so centring the
 // button itself leaves flag and code stacked inside that wrapper. The label is
 // the element that has to become the flex row.
-.language-button ::ng-deep .mdc-button__label,
-.language-button .mdc-button__label {
+//
```

Stylelint (17.14.1)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)